config/routes.rb

```diff
@@ -3,6 +3,7 @@
 
   Spina::Engine.routes.draw do
     resources :registers, only: [:show, :index]
+    get '/registers/:id', to: 'registers#index', as: 'register_index'
```

Comment:
Don't think we need this route 🤔

Reply:
I added this because what we were using previously has a `.` in it so was breaking filtering and search:
```
>>  registers_path(@register.slug)
=> "/registers.country"
```
let me know if there's a better way to fix this.

Reply:
OK I figured this out now. The issue was https://stackoverflow.com/questions/5674116/path-helpers-generate-paths-with-dots-instead-of-slashes